<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>searchInput</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="resultsList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="contextDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>551</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
